src/main/resources/templates/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<script src="/webjars/jquery/jquery.min.js"></script>
```

Comment:
These "webjars" are something I ran into while following the OAuth tutorial. They are included in the `pom.xml` and give us access to various resources hosted on our own domain. The ones I included so far are:

- bootstrap
- jquery
- js-cookie

Feel free to add more as needed.

Reply:
Very useful, nice job